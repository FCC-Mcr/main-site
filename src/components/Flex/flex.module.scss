@import "../../styles/utility/responds-to.scss";

.flex {
  display: flex;
  flex-wrap: wrap;
  > div {
    margin-bottom: 4rem;
    @include responds-to("sm", "max") {
      margin-bottom: 3rem;
    }
  }
}
