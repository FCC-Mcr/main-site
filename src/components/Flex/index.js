import React from "react"
import styles from "./flex.module.scss"

const index = ({ children }) => {
  return <div className={`${styles.grid} center px-1`}>{children}</div>
}

export default index
