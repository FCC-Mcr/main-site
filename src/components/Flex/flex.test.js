import React from "react"
import { render } from "@testing-library/react"
import "@testing-library/jest-dom/extend-expect"

import Flex from "./"

describe("Flex component", () => {
  test("Snapshot", () => {
    const { container } = render(<Flex />)
    expect(container).toMatchSnapshot()
  })
})
