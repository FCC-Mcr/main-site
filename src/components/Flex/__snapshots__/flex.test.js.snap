// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Flex component Snapshot 1`] = `
<div>
  <div
    class="flex center px-1"
  />
</div>
`;
